### PR TITLE
Refs #32880 -- Made some titles consistent in how-to documents.

### PR DESCRIPTION
**Pull request by evildmp**
Retitled documents for consistency, since some already (correctly) begin
"How to...".

### CODE REVIEW

File: docs/howto/custom-lookups.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-==============
-Custom Lookups
-==============
+===========================
+How to write custom Lookups
```
Comment (tim-mccurrach) [inline docs/howto/custom-lookups.txt:2]:
```suggestion
How to write custom lookups
```

File: docs/howto/auth-remote-user.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
-====================================
-Authentication using ``REMOTE_USER``
-====================================
+===================================================
+How to perform authentication using ``REMOTE_USER``
```
Comment (tim-mccurrach) [inline docs/howto/auth-remote-user.txt:2]:
```suggestion
How to authenticate using ``REMOTE_USER``
```
Not sure which is better here? 🤔

**Review by tim-mccurrach — APPROVED** [review]
Nice!! 👍

File: docs/howto/custom-lookups.txt (hunk 1 above)
Comment (evildmp) [reply docs/howto/custom-lookups.txt]: Agreed on both your suggestions, thanks.

**Comment by felixxm** [comment]
@evildmp Thanks :+1: Can we retitle the remaining four documents as well?

https://github.com/django/django/blob/4be58fcc369a024bd7ac72bb09af7d230499cccb/docs/howto/deployment/index.txt#L1-L3
https://github.com/django/django/blob/4be58fcc369a024bd7ac72bb09af7d230499cccb/docs/howto/static-files/index.txt#L1-L3
https://github.com/django/django/blob/4be58fcc369a024bd7ac72bb09af7d230499cccb/docs/howto/static-files/deployment.txt#L1-L3
https://github.com/django/django/blob/4be58fcc369a024bd7ac72bb09af7d230499cccb/docs/howto/deployment/wsgi/apache-auth.txt#L1-L3

**Comment by evildmp** [comment]
@felixxm Yes, will do. I'll do that in the next PR, unless it's important for them all to be done together.

**Comment by felixxm** [comment]
> @felixxm Yes, will do. I'll do that in the next PR, unless it's important for them all to be done together.

Sure, we can change them in a subsequent PR.

**Comment by felixxm** [comment]
> > @felixxm Yes, will do. I'll do that in the next PR, unless it's important for them all to be done together.
> 
> Sure, we can change them in a subsequent PR.

I prepared PR #14716.

**Pull request merged**